07-react-motivation/README.md

```diff
@@ -0,0 +1,96 @@
+
+##Why learn React? 
```

Comment:
For some reason this isn't rendering properly on the README

![image](https://user-images.githubusercontent.com/23728476/104549652-2109dc00-55e8-11eb-9dd5-9b17819515f8.png)